app/models/changeset/pull_request.rb

```diff
@@ -134,16 +134,17 @@ def message
   private
 
   def section_content(section_title, text)
+    # ### Risks or Risks followed by === / ---
     desired_header_regexp = "^(?:\\s*#+\\s*#{section_title}.*|\\s*#{section_title}.*\\n\\s*(?:-{2,}|={2,}))\\n"
-    content_regexp = '([\W\w]*?)' # capture all section content, including new lines
+    content_regexp = '([\W\w]*?)' # capture all section content, including new lines, but not next header
     next_header_regexp = '(?=^(?:\s*#+|.*\n\s*(?:-{2,}|={2,}\s*\n))|\z)'
 
     text[/#{desired_header_regexp}#{content_regexp}#{next_header_regexp}/i, 1]
   end
 
   def parse_risks(body)
     body_stripped = ActionController::Base.helpers.strip_tags(body)
-    section_content('Risks', body_stripped).to_s.strip.presence
+    section_content('Risks', body_stripped).to_s.rstrip.sub(/\A\s*\n/, "").presence
```

Comment:
explain?

Reply:
leave leading spaces to not change indentation

Reply:
i meant in a code comment.